src/components/Header/PngBalanceContent.tsx

```diff
@@ -100,7 +100,7 @@ export default function PngBalanceContent({ setShowPngBalanceModal }: { setShowP
         <CardNoise />
         <CardSection gap="md">
           <RowBetween>
-            <TYPE.white color="white">{t('header.pngBreakDown')}</TYPE.white>
+            <TYPE.white color="white">{t('header.pngBreakDown', { symbol: getPngSymbol(chainId) })}</TYPE.white>
```

Comment:
what if we keep function call on top of component like 
`const pngSymbol = getPngSymbol(chainId)`

then use `{ symbol: pngSymbol }`, so that we dont need to call `getPngSymbol` multiple times.

Please apply that in each file wherever required

Reply:
Good idea.